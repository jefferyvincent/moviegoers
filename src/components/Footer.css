.footer {
    width: 100%;
    clear: both;
    text-align: center;
    color: grey;
    margin-bottom: 2rem;
}