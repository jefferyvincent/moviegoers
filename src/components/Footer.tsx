import React from 'react';
import './Footer.css';

const Footer: React.FC = () => {
    const date = new Date();

    return (
        <footer className="footer">
            <hr />
            <div className="copyright">&copy; {date.getFullYear()} Ultimate Moviegoers Guide</div>
        </footer>
    )
}

export default Footer;