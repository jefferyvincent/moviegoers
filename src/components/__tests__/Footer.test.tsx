import React from 'react';
import {render } from 'react-testing-library';
import { shallow } from 'enzyme';

import Footer from '../Footer';

describe('<Footer/>', () => {
  let wrapper:any;
  beforeEach(() => { wrapper = shallow(<Footer />); });

  it("renders", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
  
  it('renders a footer', () => {
    expect(wrapper.find('footer.footer').length).toEqual(1);
  });

  it('renders an hr', () => {
    expect(wrapper.find('hr').length).toEqual(1);
  });
});