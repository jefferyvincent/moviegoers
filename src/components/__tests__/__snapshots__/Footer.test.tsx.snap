// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <hr />
    <div
      class="copyright"
    >
      Â© 2019 Ultimate Moviegoers Guide
    </div>
  </footer>
</DocumentFragment>
`;
